src/vcSettings.cpp

```diff
@@ -359,6 +359,7 @@ bool vcSettings_Load(vcSettings *pSettings, bool forceReset /*= false*/, vcSetti
     vcHotkey::Set(vcB_ToggleMeasureAreaTool, data.Get("keys.%s", vcHotkey::GetBindName(vcB_ToggleMeasureAreaTool)).AsInt(65));
     vcHotkey::Set(vcB_ToggleInspectionTool, data.Get("keys.%s", vcHotkey::GetBindName(vcB_ToggleInspectionTool)).AsInt(66));
     vcHotkey::Set(vcB_ToggleAnnotateTool, data.Get("keys.%s", vcHotkey::GetBindName(vcB_ToggleAnnotateTool)).AsInt(67));
+    vcHotkey::Set(vcB_ToggleMeasureHeightTool, data.Get("keys.%s", vcHotkey::GetBindName(vcB_ToggleMeasureHeightTool)).AsInt(68));
```

Comment:
F11 is Fullscreen in Chrome, so this keybinding will need to change.

@OsiJr mentioned having an idea for not using the function keys? It would also be nice to see the `SDL_SCANCODE_` values used instead of the hardcoded numbers, and where appropriate the `vcMOD_` value ideally OR'd in.

Reply:
The function keys are causing headaches everywhere but outside the scope of this MR to fix it.